radixdlt/babylon-wallet-ios
# [ABW-2482] Replace Raster Icons With PDF

Jira ticket: [ABW-2482](https://radixdlt.atlassian.net/browse/ABW-2482)

## Description
We still had some raster icons in the app, that didn't exist as vector images before, they have now been replaced.
- Fungible token placeholder icon
- NFT icon
- XRD icon
- Poolunit icon

## How to test
Look through all places where these icons appear and make sure they load properly and look like before.

## Screenshot
![image](https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/e5c02d60-77b1-4dad-81a8-dc091fc40925)

## PR submission checklist


[ABW-2482]: https://radixdlt.atlassian.net/browse/ABW-2482?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
